### PR TITLE
Fix Env Variables

### PR DESCRIPTION
**Pull request by mburkert3**
I had `SONATYPE_USERNAME `and `SONATYE_PASSWORD` backwards. Also changed gradle.properties to version 3.0.0 for release

### CODE REVIEW

**Review by shaikhu — APPROVED** [review]

**Review by bhamail — APPROVED** [review]
+1 Nice and clean

File: build.gradle [hunk 1]
```diff
@@ -1,5 +1,6 @@
 plugins {
   id 'java-gradle-plugin'
+  id 'java-library'
```
Comment (bhamail) [inline build.gradle:3]: Just curious: Why was this line added?

**Review by bhamail — COMMENTED** [review]
Quick question inline

File: build.gradle (hunk 1 above)
Comment (mburkert3) [reply build.gradle]: Added it locally to test something but forgot to remove it. [Commit](https://github.com/sonatype-nexus-community/scan-gradle-plugin/pull/182/commits/4753ad0fc473a544d3765a435113d42fbc9ce72c)
